### ISSUE TITLE
Windows Management Instrumentation swallowing CPU resources

### ISSUE THREAD

**Issue by MeowMeowKins** [amidaware/tacticalrmm#1488]
**Server Info (please complete the following information):**
 - OS: Debian 10
 - Browser: Chrome,Firefox
 - RMM Version (as shown in top left of web UI): v0.15.9

**Installation Method:**
  - [x] Standard
  - [ ] Docker

**Agent Info (please complete the following information):**
- Agent version (as shown in the 'Summary' tab of the agent from web UI): v2.4.6
- Agent OS: Windows Server 2019,2022

**Describe the bug**
When the agent decides to fetch statistics it seems that the WMI Instrumentals starts using all CPU available and does not stop until the Windows Management Instrumentation service is restarted.

**To Reproduce**
1. Install TacticalRMM on a brand new ISO for Server 2019 or 2022.
2. Use the server like you normally would.
3. Wait until you see the process spike in Task Manager, Windows Management Instrumentation is now using almost all CPU resources.

**Expected behavior**
Expecting peaks in WMI process, which is understandable.

**Screenshots**
![WMI process usage](https://user-images.githubusercontent.com/22405890/232708917-16b6f315-2db1-488e-ba54-1218724906f7.png)

**Additional context**
We are using the Windows Servers with Remote Desktop Services, we have experienced this happening in 3 user environments and 80+ user environments. We now have a task that restarts WMI when it starts using 90> CPU for a couple minutes.

**Comment by silversword411** [comment]
> fetch statistics

What do you mean by this?

This sounds like a WMI bug, not a TRMM bug

**Comment by MeowMeowKins** [comment]
Yeah, sorry, it does seem like a WMI bug, with the TacticalRMM agent turned off for the day it started happening as well.
Sorry for wasting your time!
